Packs/IllusiveNetworks/Integrations/IllusiveNetworks/IllusiveNetworks.yml

```diff
@@ -516,7 +519,133 @@ script:
     - contextPath: Illusive.Event.status
       description: The status command ( Done, InProgress)
       type: String
-  dockerimage: demisto/python3:3.8.2.6981
+  - arguments:
+    - default: false
+      description: The incident ID on which to retrieve the associated events
+      isArray: false
+      name: incident_id
```

Comment:
Where is this ID can be retrieved from? Please edit the description to include the command to run to get the ID from